cwd/app.js

```diff
@@ -217,15 +220,26 @@
   function startKiosk(duration) {
     let views = allGlyphs.filter(obj => obj.view).map(obj => obj.view);
     let index = 0;
+    let hashes = ['#glsc', '#lake', '#air'];
```

Comment:
The hardcoding here is not ideal.  I initially thought it was simply something that we would make easier in the database later, but note line 221.  You already have a list of view objects, so you can probably write a map function that will turn this list of view objects into the string array you have here.  That way, we don't hardcode it.  I'd challenge you to do this.